src/modules/core/index.ts

```diff
@@ -96,7 +98,17 @@ function init(config: ServiceWorkerConfiguration = {}) {
   });
 }
 
+function installNoOpServiceWorker() {
+  import(/* webpackChunkName: "service-worker-remover" */ '../service-worker-remover/index').then(
+    async ({ ServiceWorkerRemover }) => {
+      const swRemover = new ServiceWorkerRemover();
+      swRemover.installNoOpServiceWorker();
```

Comment:
Not using the value elsewhere, so you can just...

`new ServiceWorkerRemover().installNoOpServiceWorker();`